spikeinterface/core/baserecording.py

```diff
@@ -136,7 +136,9 @@ def _save(self, format='binary', **save_kwargs):
             # TODO save propreties as npz!!!!!
             folder = save_kwargs['folder']
             file_paths = [folder / f'traces_cached_seg{i}.raw' for i in range(self.get_num_segments())]
-            dtype = save_kwargs.get('dtype', 'float32')
+            dtype = save_kwargs.get('dtype', None)
+            if dtype is None:
+                dtype = self.get_dtype()
```

Comment:
@samuelgarcia I found out that save did not keep the recording dtype if not specifically passed as an argument! I believe this is the correct behavior. If not specified, use the one from the recording

Reply:
outch. I think I did this at the beggining of the new API to be sure that it works.
thank you for catching.